templates/jenkins.j2

```diff
@@ -0,0 +1,33 @@
+NAME={{ jenkins_name }}
+JAVA={{ jenkins_java_location }}
+JAVA_ARGS="{{ (jenkins_java_opts + jenkins_java_user_opts) | join(" ") }}"
+PIDFILE={{ jenkins_pidfile_path }}
+JENKINS_USER=$NAME
+JENKINS_GROUP=$NAME
+JENKINS_WAR={{ jenkins_war_path }}
+JENKINS_HOME={{ jenkins_home_path }}
+
+RUN_STANDALONE={{ jenkins_run_standalone }}
+
+JENKINS_LOG={{ jenkins_log }}
+MAXOPENFILES=8192
+{% if jenkins_set_umask %}
```

Comment:
I think that defaulting to `UMASK=022` is fine here. The user can override `jenkins_umask` directly without need for the `jenkins_set_umask` attribute.